services/core-web/src/components/mine/Permit/MinePermitTable.tsx

```diff
@@ -313,6 +324,17 @@ const columns: ColumnsType<MinePermitTableItem> = [
               </button>
             </div>
           </AuthorizationWrapper>
+          {isFeatureEnabled(Feature.DIGITIZED_PERMITS) && (
+            <div className="custom-menu-item">
+              <button
```

Comment:
I notice that this is a not-antd button! Curious if it was on purpose (the custom-menu-item hints that it shouldn't look like a normal button)

Reply:
Yeah, this is really just a menu link that is formatted into the existing menu.  Using an antd button does some things that make it look pretty janky.

There's a plan to replace this menu in the future, so I think we can revisit the display when that happens.